### PR TITLE
Release 5.1.1

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: docs/appendices/release-notes/5.1.1.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+=============
+
+Released on 2022-11-09.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 4.0.2 or higher
+    before you upgrade to 5.1.1.
+
+    We recommend that you upgrade to the latest 4.8 release before moving to
```
Comment (BaurzhanSakhariev) [inline docs/appendices/release-notes/5.1.1.rst:14]: latest 5.0?

File: docs/appendices/release-notes/5.1.1.rst [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+See the :ref:`version_5.0.0` release notes for a full list of changes in the
+5.0 series.
```
Comment (BaurzhanSakhariev) [inline docs/appendices/release-notes/5.1.1.rst:39]: 5.1?

**Review by BaurzhanSakhariev — APPROVED** [review]
looks like there are some minor typos, otherwise LGTM

**Comment by matriv** [comment]
> looks there are some minor typos, otherwise LGTM

Thx! can you double check once more please?

**Pull request merged**